loops-arrays-objects.js

```diff
@@ -74,18 +101,23 @@ const NOVIEmployees = [
 //   { firstName: 'Mark', lastName: 'Rensen', email: 'Mark.Rensen@novi.nl' },
 //  ];
 // ==========================================
+// for (let i = 0; i < NOVIEmployees.length ; i++) {
+//
+// }
 
+console.log('***********************')
 
+for (let i = 0; i < NOVIEmployees.length ; i++) {
+    const employee = NOVIEmployees[i];  //vraag > waarom mag je hier een nieuwe variabele aan?
```

Comment:
Jouw vraag: 'Waarom maak je hier een variabele aan?':

- Je plaatst deze variabele binnen de {  } (Snorretjes, zoals Nova dat noemt. Ofwel accolades) - Binnen deze snorretjes is een zogenaamde: Scope.  
https://www.w3schools.com/js/js_scope.asp

- De for-loop zorgt ervoor dat er elke ronde/iteratie een nieuwe waarde in deze variabele komt:

Lus ronde 0: 
const employee = NOVIEmployees[0]; ==> Nova
Lus ronde 1:
const employee = NOVIEmployees[0]; ==> Sam
Lus ronde 2:
const employee = NOVIEmployees[0]; ==> Tessa
Lus ronde 3:
const employee = NOVIEmployees[0]; ==> Mark

Daarna kun je d.m.v. zogenaamde 'dot notatie' de email van deze specifieke employee onttrekken.

Dot notatie: employee.email

employee zou je kunnen vergelijken binnen een regel/record binnen Excel. 

Daarna ken je de bewerkte email toe aan deze employee: employee.email = etc.